### PR TITLE
22-4-22

### PR DESCRIPTION
**Pull request by umaaug**

### CODE REVIEW

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md [hunk 1]
```diff
@@ -362,7 +359,8 @@ who had not kept sacred firas. 497 The Pār, gr, further provides that all relat
...
+
+In modern times the procedure of cremation is generally on the lines of that described in the Asy. gr, cited above and the procedure described in the Garuḍapurāpa II, 4, 41 ff. The following is a summary of the procedure set out in the Antyesti paddhati of Narāyanabhatta which is generally followed by Ṛgveding in Western India. The dead body is washed with water, is decked with a talaka on the forehead, is clothed in a garment, is covered with a paste of Jatāmāmsi and a garland of naladas is placed round its neck, Then the performer washes his own fest, sips water, performs prādāyāma without Vedic mantra, then makes a Saṅkalpa (as in note 500 ),500 he wears his sacred thread under the left shoulder (and on the right shoulder) and prays to the earth with the mantra 'upasarpa' (Ṛg. X. 18.10), he then thrice goes round the place dug up (for cremation) and sprinkles water with a sami branch at the end of the mantra
```
Comment (vvasuki) [inline kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:363]: tilaka on the forehead, feet etc.

Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:363]: This paragraph is not proof-read yet. I closed with footnote 493.

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md [hunk 2]
```diff
@@ -336,53 +336,40 @@ H. D. 27
...
 
- _ ___494, अधानमिचितः। एतदेव भूनिजोपणमेतत्सनान कर्न यदन्यदग्निकर्मण.। Taquar. XIII 8,4,11
+[^494]: अधानग्निचितः। एतदेव भूमिजोपणमेतत्सनान कर्म यदन्यदग्निकर्मण.। शतपथवा . XIII 8,4,11
 
-495 आहिताग्निर्यथान्यायं दग्धन्यस्त्रिंभिरनिभिः। अनादितासिरेकेन लौकिकेनेतरो Gat:ll TCT G. by FÆGT. ON T III 2, 3 p 873 This is a garot (JERTET) 23 67
+[^495]: आहिताग्निर्यथान्यायं दग्धन्यस्त्रिंभिरनिभिः। अनाहिताग्निरेकेन लौकिकेनेतरोज्ंनः ॥ वृद्धया q. by मिता. ON या  III 2, अपरार्क  p 873. This is कूर्मपुराण (उत्तरार्ध) 23 67
```
Comment (vvasuki) [inline kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:343]: लौकिकेनेतरो जनः

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md [hunk 3]
```diff
@@ -336,53 +336,40 @@ H. D. 27
...
 
- _ ___494, अधानमिचितः। एतदेव भूनिजोपणमेतत्सनान कर्न यदन्यदग्निकर्मण.। Taquar. XIII 8,4,11
+[^494]: अधानग्निचितः। एतदेव भूमिजोपणमेतत्सनान कर्म यदन्यदग्निकर्मण.। शतपथवा . XIII 8,4,11
```
Comment (vvasuki) [inline kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:341]: plz send image for this - doubtful

Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:341]: image

Comment (vvasuki) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:341]: I see several mistakes. First word - अथानग्निचितः। 

Please use new pdf I sent - much clearer. Please send image from there - I'll show a trick by which you can make correction much easier.

Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:341]: image
This is from new PDF.

Comment (vvasuki) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:341]: No - this is not the new pdf. The below is from the new pdf. See the difference? Can you make sure you got the correct pdf?

image

**Pull request merged**